lib/hobo_fields/model/index_spec.rb

```diff
@@ -54,22 +54,22 @@ def primary_key?
       end
 
       def default_name?
-        name == @model.connection.index_name(table, :column => fields)
+        name == @model.connection.index_name(table, column: fields)
       end
 
       def to_add_statement(new_table_name, existing_primary_key = nil)
         if primary_key? && !ActiveRecord::Base.connection.class.name.match?(/SQLite3Adapter/)
           to_add_primary_key_statement(new_table_name, existing_primary_key)
         else
-          r = "add_index :#{new_table_name}, #{fields.map(&:to_sym).inspect}"
-          r += ", :unique => true" if unique
-          r += ", :where => '#{self.where}'" if self.where.present?
+          r = +"add_index :#{new_table_name}, #{fields.map(&:to_sym).inspect}"
+          r << ", unique: true" if unique
+          r << ", where: '#{self.where}'" if self.where.present?
```

Comment:
Not related, but I learned of the `+` notation above after I'd converted this code to work with frozen string literals. So I switched over to it here, returning to using `<<` to keep adding to the buffer.
What's up with `check_name` below?! It's not used. It feels like a merge error sometime in the past?

Reply:
I resolved this and it simpler still.